Makefile

```diff
@@ -81,6 +81,10 @@ push:
 	git commit --quiet -m "Deploy to GitHub Pages";\
 	git push --force "git@github.com:cdecker/lightning-integration.git" master:gh-pages
 
+docker-build:
+	docker build --tag=lnintegration .
+docker-run: docker-build
```

Comment:
Notably this will always build the docker image even if it is up to date.

Reply:
If it is up to date, the build should cost almost nothing because docker should have everything cached already, no?

Reply:
Right, it was just an observation :wink: